.github/workflows/push.yml

```diff
@@ -191,3 +190,40 @@ jobs:
           tag: ${{ github.ref }}
           overwrite: true
           file_glob: true
+      - name: create homebrew formulae
+        env:
+          TARGET_VERSION: ${{ needs.set_version.outputs.version }}
+        run: |
+          mkdir homebrew
+          export ARM_TARGET_SHA256=$(ls release/signed/*darwin_arm64.zip | sha256sum | cut -f 1 -d ' ' )
+          export AMD_TARGET_SHA256=$(ls release/signed/*darwin_amd64.zip | sha256sum | cut -f 1 -d ' ')
+          envsubst < homebrew.template.rb > homebrew/keboola-as-code.rb
+          cat homebrew/keboola-as-code.rb
+      - name: Upload homebrew formulae to artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: homebrew
+          path: homebrew
+
+  release-homebrew:
+    needs:
+      - release-mac-os
+    runs-on: ubuntu-latest
```

Comment:
musel som to dat do separatneho tasku pretoze na macu sa neda pustit `github-action-push-to-another-repository@main`